tests/integration/query/commits/simple_test.go

```diff
@@ -389,7 +389,7 @@ func TestQuery_CommitsWithAllFieldsWithUpdate_NoError(t *testing.T) {
 							"links":        []map[string]any{},
 						},
 						{
-							"cid":          "bafyreigtnj6ntulcilkmin4pgukjwv3nwglqpiiyddz3dyfexdbltze7sy",
+							"cid":          testUtils.NewUniqueCid("name create"),
```

Comment:
question: don't you think using `int` would be easier to read. At first I though these strings have some meaning (maybe you have predefined list of "composite", "update" ...) but reviewing further I learning it's just an arbitrary unique id. I think `int` is easier to use here and less prone to spelling errors.

Reply:
I definitely understand what you mean, and it amused me a little to see this suggestion after your strongly typed id suggestion.  

A string here is less information dense than an int, but typo's dont actually matter too much here - as if the same cid is referenced twice with different strings the test will fail (this does expose us to accidentally passing bad production code though if cids should be the same but differ).

Another option would be an enum, or variant of,that seems a bit overkill though:
```
func NewUniqueCid(keyVals ...any) *UniqueCid { ... }

testUtils.NewUniqueCid(testUtils.CreateCid, "age")
```

Reply:
I'm not a fan of the enum suggestion. I don't mind the string. I do agree that `int` would be less error prone but strings are more descriptive (as descriptive as the test author wants).